tests/test_dists.py

```diff
@@ -31,6 +32,25 @@ def test_polyagamma():
     assert res1.eval() == res2.eval()
 
 
+
+def test_multichoice():
+
+    rng1 = at.random.RandomStream(12345)
+    p_val = np.stack([np.repeat(0.1, 10)] * 3, axis=0)
+    res1 = rng1.gen(multichoice, np.array(10, dtype = np.int64), 1, True, p_val)
+    
+    rng2 = at.random.RandomStream(12345)
+    res2 = rng2.gen(multichoice, np.array(10, dtype = np.int64), 1, True, p_val)
+    # add assersion fpr results
+    assert np.alltrue(res1.eval() == res2.eval())
```

Comment:
The graphs `res1` and `res2` are literally the same, no?  It's not clear what this `assert` is supposed to be testing, and, as the comment says, we absolutely need to confirm that this is working before considering the results of a model that uses this sampler.